code/frameworks/sveltekit/src/preset.ts

```diff
@@ -18,8 +18,7 @@ export const viteFinal: NonNullable<StorybookConfig['viteFinal']> = async (confi
   plugins = await withoutVitePlugins(plugins, [
     // pre @sveltejs/kit@1.0.0-next.574
```

Comment:
I wouldn't worry supporting anything pre-1.0

Reply:
you're right, we should remove that now. I believe this was written before the 1.0 launch.